docs/model/product-searching.md

```diff
@@ -0,0 +1,31 @@
+# Product Searching
```

Comment:
There is no intro for product searching in general. There should be info, that we have a product search in admin (classic SQL) and on FE (elastic). Or you can change the name of the article to Front-end Product Search.

Reply:
renamed to Front-end Product Searching (including file name and links to this file)